src/components/ReactTable/ReactTable.tsx

```diff
@@ -31,7 +31,11 @@ export const ReactTable = ({
     return rows.map(row => {
       prepareRow(row)
       return (
-        <Tr {...row.getRowProps()} tabIndex={row.index} onClick={() => onRowClick && onRowClick()}>
+        <Tr
+          {...row.getRowProps()}
+          tabIndex={row.index}
+          onClick={() => onRowClick && onRowClick(row)}
```

Comment:
We should note in risks that this change also affects Defi section as well as dashboard accounts and they should be regression tested.